std_execution.bs

```diff
@@ -2950,79 +2944,90 @@ namespace std::execution {
     template &lt;class CPO>
     struct get_completion_scheduler_t;
 
+    struct schedule_t;
+    struct transfer_just_t;
+    struct on_t;
+    struct transfer_t;
+    struct schedule_from_t;
+    struct then_t;
+    struct upon_error_t;
+    struct upon_done_t;
+    struct let_value_t;
+    struct let_error_t;
+    struct let_done_t;
+    struct bulk_t;
+    struct split_t;
+    struct when_all_t;
+    struct when_all_with_variant_t;
+    struct transfer_when_all_t;
+    struct transfer_when_all_with_variant_t;
+    struct done_as_optional_t;
+    struct done_as_error_t;
+    struct ensure_started_t;
+    struct start_detached_t;
+  }
+  inline namespace <i>unspecified-inline</i> {
+    // [execution.senders.connect], the connect sender algorithm
+    inline constexpr <i>unspecified</i>::connect_t connect{};
+
+    template &lt;class S, class R>
+      using connect_result_t = decltype(connect(declval&lt;S>(), declval&lt;R>()));
+
+    // [execution.senders.queries], sender queries
+    inline constexpr <i>unspecified</i>::forwarding_sender_query_t forwarding_sender_query{};
+
     template &lt;class CPO>
-    inline constexpr get_completion_scheduler_t&lt;CPO> get_completion_scheduler{};
+    inline constexpr <i>unspecified</i>::get_completion_scheduler_t&lt;CPO> get_completion_scheduler{};
 
     // [execution.senders.factories], sender factories
-    struct schedule_t;
-    inline constexpr schedule_t schedule{};
+    inline constexpr <i>unspecified</i>::schedule_t schedule{};
     template&lt;<i>movable-value</i>... Ts>
       <i>unspecified</i> just(Ts &&...) noexcept(<i>see below</i>);
     template&lt;<i>movable-value</i> Error>
       <i>unspecified</i> just_error(Error &&) noexcept(<i>see below</i>);
     <i>unspecified</i> just_done() noexcept;
     template&lt;<i>movable-value</i>... Ts>
       <i>unspecified</i> just(Ts &&...);
```

Comment:
Since these are functions (not CPOs) and we don't want them findable with ADL, please put them into their own (non-`inline`) namespace.